### ISSUE TITLE
Exception inside subscription is silently ignored.

### ISSUE THREAD

**Issue by nojaf** [fable-elmish/elmish#124]
### Description

If something goes wrong inside a subscription function, it is never displayed in the browser console.

### Repro code

```fsharp
module App.View

open Elmish
open Elmish.Browser.Navigation
open Elmish.Browser.UrlParser
open Fable.Core
open Fable.Core.JsInterop
open Fable.Import
open Fable.Import.Browser
open Fable.Helpers.React
open Fable.Helpers.React.Props
open System

type Model = int

type Msg =
    | Fire
    | SupaHot

[<Emit("foo.bar")>]
let interoptFn:int = jsNative

let fireInTheHole (dispatch:Dispatch<Msg>) =
    let res = interoptFn
    dispatch SupaHot


let init () =
    0, Cmd.none

let update msg model =
    match msg with
    | Fire ->
        model, Cmd.ofSub fireInTheHole
    | SupaHot ->
        9001, Cmd.none

let view model dispatch =
    div [] [
        label [] [sprintf "%i" model |> str]
        br []
        button [OnClick (fun _ -> dispatch Fire)] [str "Fire"]
    ]

open Elmish.React
open Elmish.Debug
open Elmish.HMR

// App
Program.mkProgram init update view
#if DEBUG
|> Program.withDebugger
|> Program.withHMR
#endif
|> Program.withReact "elmish-app"
|> Program.run
```

### Expected and actual results

It would be nice if you saw the same error as you would type in `foo.bar` in the console log.
In this case `Uncaught ReferenceError: foo is not defined at ...`

### Related information 

* fable-elmish version: 0.9.2
* fable-compiler version: 1.2.4
* fable-core version: 1.2.4
* Operating system: Windows 7 with chrome 64.

**Comment by et1975** [comment]
Makes sense, thanks for the report!

**Comment by et1975** [comment]
I think I misread the report, the situation as described is actually handled correctly - the exception is handled by the dispatch loop and reported with `onError`. Perhaps you mean subscriptions that are passed into `withSubscription`? Or maybe you see the `withDebugger` taking over the error reporting and swallowing the errors [that just got fixed](/fable-elmish/debugger/pull/4)?

**Comment by nojaf** [comment]
Well if something goes wrong in the subscription I would expect it to show up in the console.log. But if you say the dispatch loop catches the error then it makes I'm not seeing anything. How does one use this `onError` reporting? I'll also try the new debugger once it has been released.

**Comment by et1975** [comment]
You don't use `onError`, elmish core does, but the point is I tried your sample (thanks for that btw) and it works as expected: the exception is logged into console. I did however strip out all the debugger stuff, which makes me think what you are experiencing is the debugger issue that just got fixed. Initial subscription throwing and the debugger fixes are released in fable-elmish@1.0.1 and fable-elmish-debugger@1.0.1, please give them a spin.

**Comment by nojaf** [comment]
Tried the `1.0.1` versions, it works as expected. I see my error inside the console log as well as inside the debug tools. Thank you.
